### PR TITLE
xy <-> yx for keypoints

### PR DESCRIPTION
**Pull request by yuyu2172**
Merge after #234.

### CODE REVIEW

File: chainercv/datasets/cub/cub_keypoint_dataset.py [hunk 1]
```diff
@@ -93,8 +94,9 @@ def get_example(self, i):
 
         if self.crop_bbox:
             bbox = self.bboxes[i]  # (x, y, width, height)
-            img = img[bbox[1]: bbox[1] + bbox[3], bbox[0]: bbox[0] + bbox[2]]
-            keypoint[:, :2] = keypoint[:, :2] - np.array([bbox[0], bbox[1]])
+            img =\
+                img[:, bbox[1]: bbox[1] + bbox[3], bbox[0]: bbox[0] + bbox[2]]
```
Comment (Hakuyume) [inline chainercv/datasets/cub/cub_keypoint_dataset.py]: It is better to make another PR for this bug fix. It should be sent for `master` branch.

Comment (yuyu2172) [inline chainercv/datasets/cub/cub_keypoint_dataset.py]: https://github.com/chainer/chainercv/pull/248

**Comment by Hakuyume** [comment]
I'm sorry, I missed this line (I should point it out in WH -> HW)
https://github.com/yuyu2172/chainercv/blob/09df2ddd6a627b97c6f17938241bbb2ae94fba8e/chainercv/evaluations/eval_pck.py#L15

File: chainercv/transforms/keypoint/translate_keypoint.py [hunk 2]
```diff
@@ -1,19 +1,18 @@
-def translate_keypoint(keypoint, x_offset=0, y_offset=0):
+def translate_keypoint(keypoint, y_offset=0, x_offset=0):
     """Translate keypoints.
 
     This method is mainly used together with image transforms, such as padding
     and cropping, which translates the left top point of the image from
```
Comment (Hakuyume) [inline chainercv/transforms/keypoint/translate_keypoint.py]: `left top` -> `top left`?

**Comment by Hakuyume** [comment]
LGTM
